samples/tmo_shell/prj.conf

```diff
@@ -160,3 +160,4 @@ CONFIG_LOG_PRINTK=n
 CONFIG_TMO_TEST_MFG_CHECK_GOLDEN=y
 CONFIG_TMO_TEST_MFG_CHECK_ACCESS_CODE=y
 CONFIG_DFU_GECKO_LIB=y
+CONFIG_ADC_GECKO=y
```

Comment:
something is not quite right here. maybe a \n is needed at the end of this line

Reply:
That is exactly the problem, there should be blank lines at the end of files